forwarder.go

```diff
@@ -10,24 +10,44 @@ import (
 	"net/http"
 	"os"
 	"strings"
+	"sync"
 )
 
+const workers = 8
+
 func main() {
 	log.Println("Splunk forwarder: Started")
 	defer log.Println("Splunk forwarder: Stopped")
 
+	forSplunk := make(chan string)
```

Comment:
I think the name of this variable is not very significative. Maybe it should be `logStream` or `logChan` .